assignments/sql-1.md

```diff
@@ -5,4 +5,15 @@ language: SQL
 exercises: ['QAQC 1', 'Importing Data', 'Creating Tables', 'Adding Records', 'Updating Records', 'Basic Queries', 'Filtering', 'Sorting', 'Distinct', 'Missing Data', 'Grouping', 'Aggregation 1', 'Aggregation 2']
 ---
 
+####Learning Objectives
+
+> Following this assignment students should be able to:
+
+> - set-up an Excel spreadsheet for quality control at data entry
```

Comment:
how about a little broader: "implement quality control for data entry in spreadsheets"